app/models/user.rb

```diff
@@ -0,0 +1,75 @@
+class User < ActiveRecord::Base
+  has_secure_password
+
+  has_many :sent_matches, class_name: "Match"
+  has_many :received_matches, class_name: "Match", foreign_key: :matched_user_id
+  has_many :matched_users, through: :matches
+  has_one :business, foreign_key: :owner_id
+  has_many :written_reviews, class_name: "Review", foreign_key: :reviewer_id, dependent: :destroy
+  has_many :received_reviews, class_name: "Review", foreign_key: :reviewee_id, dependent: :destroy
+  has_many :written_messages, class_name: "Conversation", foreign_key: :sender_id, dependent: :destroy
+  has_many :received_messages, class_name: "Conversation", foreign_key: :recipient_id, dependent: :destroy
+
+  has_attached_file :avatar, styles: {
+                                        thumb: '100x100>',
+                                        square: '200x200#',
+                                        medium: '300x300>'
+                                      }
+
+  validates_attachment_content_type :avatar, :content_type => /\Aimage\/.*\Z/
+
+  validates_attachment_size :avatar, :less_than => 5.megabytes
+
+   validates :first_name, presence: true
+   validates :uid, uniqueness: { scope: :provider }, on: :create
+   validates :uid, presence: true, on: :create
+
+   def matched?(user)
+      r_matches = self.received_matches.where(user: user)
+      s_matches = self.sent_matches.where(matched_user: user)
+      if r_matches != [] || s_matches != []
```

Comment:
You could just do `r_matches != [] || s_matches != []1` as the last line and that will return true/false without the if/else